core-web/libs/data-access/src/lib/dot-upload-file/dot-upload-file.service.ts

```diff
@@ -89,4 +89,22 @@ export class DotUploadFileService {
             })
         );
     }
+
+    uploadDotAsset(file: File): Observable<DotCMSContentlet> {
+        const formData = new FormData();
+        formData.append('file', file);
+        formData.append(
+            'json',
+            JSON.stringify({
+                contentlet: {
+                    file: file.name,
+                    contentType: 'dotAsset'
+                }
+            })
+        );
+
+        return this.http
+            .put<DotCMSContentlet>(`/api/v1/workflow/actions/default/fire/NEW`, formData)
```

Comment:
use a constant for this endpoint

Reply:
should we use the `DotWorkflowActionsFireService` here, `newContentlet`.